MCP_JSON_CONFIGURATION_GUIDE.md

```
@@ -0,0 +1,571 @@
+
+## Server Configuration Details
+
+### 1. HTTP Direct Transport (Recommended for Postman/Web)
+
+```jsonc
+"cqrspattern-http-direct": {
+  "type": "http",
+  "url": "http://localhost:5001/mcp",
+  "description": "CQRS Pattern API MCP Endpoint - Direct HTTP Transport",
+  "headers": {
+    "Content-Type": "application/json"
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)